### ISSUE TITLE
Kindly Add Support for Mzaalo

### ISSUE THREAD

**Issue by ArgX11** [yt-dlp/yt-dlp#7146]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

India

### Example URLs

https://www.mzaalo.com/
https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata

### Provide a description that is worded well enough to be understood

Kindly add support for this OTT website. Thanks in advance 

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
D:\Softwares\YT-DLP>ytdlp -vU https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata
[debug] Command-line config: ['-vU', 'https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-109851-ge506ea3ce1-20230212 (setts), ffprobe N-109851-ge506ea3ce1-20230212
[debug] Optional libraries: Cryptodome-3.17, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
Current Build Hash: 5590c57bd0433ed239a2deaaf92e2ad6f37fe50f53664c821575cafe106a9421
yt-dlp is up to date (stable@2023.03.04)
[generic] Extracting URL: https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata
[generic] Nil-Battey-Sannata: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] Nil-Battey-Sannata: Extracting information
[debug] Looking for embeds
ERROR: Unsupported URL: https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1518, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1594, in __extract_info
  File "yt_dlp\extractor\common.py", line 694, in extract
  File "yt_dlp\extractor\generic.py", line 2510, in _real_extract
yt_dlp.utils.UnsupportedError: Unsupported URL: https://www.mzaalo.com/watch/movie/ca785956-1c85-435e-8233-56320148c78c/Nil-Battey-Sannata
```

**Comment by ItzMaxTV** [comment]
Currently working on it..

**Comment by bashonly** [comment]
Closed by dc3c44f349ba85af320e706e2a27ad81a78b1c6e

**Comment by ArgX11** [comment]
The issue hasn't been resolved yet but surprisingly it is closed. Mzaalo.com is still not supported

```shell

D:\Softwares\YT-DLP>ytdlp -vU https://mzaalo.com/watch/MOVIE/389c892d-0b65-4019-bf73-d4edcb1c014f/Chalo-Dilli
[debug] Command-line config: ['-vU', 'https://mzaalo.com/watch/MOVIE/389c892d-0b65-4019-bf73-d4edcb1c014f/Chalo-Dilli']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-110973-g3cfd85ec20-20230601 (setts), ffprobe N-110973-g3cfd85ec20-20230601
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[generic] Extracting URL: https://mzaalo.com/watch/MOVIE/389c892d-0b65-4019-bf73-d4edcb1c014f/Chalo-Dilli
[generic] Chalo-Dilli: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] Chalo-Dilli: Extracting information
[debug] Looking for embeds
ERROR: Unsupported URL: https://mzaalo.com/watch/MOVIE/389c892d-0b65-4019-bf73-d4edcb1c014f/Chalo-Dilli
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1560, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1688, in __extract_info
  File "yt_dlp\extractor\common.py", line 710, in extract
  File "yt_dlp\extractor\generic.py", line 2568, in _real_extract
yt_dlp.utils.UnsupportedError: Unsupported URL: https://mzaalo.com/watch/MOVIE/389c892d-0b65-4019-bf73-d4edcb1c014f/Chalo-Dilli
```

**Comment by bashonly** [comment]
@ArgX11 fixed by d7aee8e310b2c4f21d50aac0b420e1b3abde21a4 -- run `yt-dlp --update-to nightly`

**Comment by ArgX11** [comment]
Thank you for the solution. Really appreciate!
